packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -753,7 +763,13 @@
                     "type": "cwsprChatFullResponseLatency"
                 },
                 {
-                    "type": "cwsprTimeToFirstDisplay"
+                    "type": "cwsprChatTimeFirstUsableChunk"
+                },
+                {
+                    "type": "cwsprChatFullServerResponseLatency"
```

Comment:
is something blocking these from being upstreamed to https://github.com/aws/aws-toolkit-common ?